hawaii-international-conference-on-system-sciences-proceedings.csl

```diff
@@ -49,13 +57,13 @@
     </date>
   </macro>
   <macro name="journal">
-    <group suffix=", " delimiter=" " font-style="italic">
-      <text variable="container-title"/>
-      <text variable="volume"/>
+    <group font-style="normal" suffix=", ">
+      <text variable="container-title" font-style="italic"/>
```

Comment:
Same here -- any indication of italics somewhere other than the styleguide?

Reply:
same as above